File: 프로그래머스/lv0/120834. 외계행성의 나이/외계행성의 나이.java
class Solution {
    public String solution(int age) {
        String answer = "";
        while (age > 0) {
            int temp = age % 10 + 'a';
            answer += String.valueOf((char) temp);
            age /= 10;
        }
        
        StringBuffer sb = new StringBuffer(answer);
        
        return sb.reverse().toString();
    }
}

File: C/프로그래머스/0/120834. 외계행성의 나이/외계행성의 나이.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

char* solution(int age) {
    // return 값은 malloc 등 동적 할당을 사용해주세요. 할당 길이는 상황에 맞게 변경해주세요.
    char buffer[4];
    int count = 0;
    
    while(age > 0){
        buffer[count++] = age % 10 + 'a';
        age /= 10;
    }
    char temp[count];
    char* answer = (char*)malloc(sizeof(char) * (count + 1));
    for(int i = 0; i < count; i++) {
        answer[i] = buffer[count - 1 - i];
    }
    answer[count] = '\0';
    return answer;
}